src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const total = days * price;
+
+  if (days >= 7) {
+    return total - 50;
```

Comment:
`avoid magic numbers` - fix it now in the whole solution and for the future. If you meet such a comment, it means that all numbers that do not carry anything semantic in the code should be made into constants whose name will explain what these numbers mean

E.g `const threeDayRentalDiscount = 20;`